evaluation/benchmarks/for-loops/aliases/1.tomp3.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# tag: wav-to-mp3
+set -e
+PASH_TOP=/home/tammam1/pash
```

Comment:
Setup `PASH_TOP` conditionally too, similarly to how IN and OUT are set below, i.e., only set it to `/home/tammam1/pash` 🙂 if it is not already set (or not set it to your local path at all).

Reply:
Ok, I removed it. Ideally, this would be set by pash, right? I think I had to add it because it wasn't being registered but I will remove it for now.